.github/workflows/ci.yml

```
@@ -18,6 +18,9 @@ jobs:
           - '3.8'
           - '3.9'
           - 'pypy3'
+        coverage:
+          - '5.5'
+          - '6.0'
     name: Python ${{ matrix.python }}
```

You could mention the coverage version in the name:

```suggestion
    name: Python ${{ matrix.python }} on coverage ${{ matrix.coverage }}
```